src/Extension.php

```diff
@@ -599,6 +617,125 @@ public function rcp_membership_payment_profile_cancelled( $success, $gateway, $g
 		return true;
 	}
 
+	/**
+	 * Can update membership billing.
+	 *
+	 * @param bool $can_update        Whether or not billing can be updated.
+	 * @param int  $rcp_membership_id ID of the membership.
+	 * @return bool
+	 */
+	public function rcp_membership_can_update_billing( $can_update, $rcp_membership_id ) {
+		$subscriptions = \get_pronamic_subscriptions_by_source( 'rcp_membership', $rcp_membership_id );
+
+		if ( 0 !== count( $subscriptions ) ) {
+			$can_update = true;
+		}
+
+		return $can_update;
+	}
+
+	/**
+	 * Subscription action links HTML.
+	 *
+	 * @param string         $actions        Formatted HTML links.
+	 * @param array          $links          Links.
+	 * @param int            $user_id        Current user ID.
+	 * @param RCP_Membership $rcp_membership Membership object.
+	 * @return string
+	 */
+	public function rcp_subscription_details_actions( $actions, $links, $user_id, $rcp_membership ) {
+		$subscriptions = \get_pronamic_subscriptions_by_source( 'rcp_membership', $rcp_membership->get_id() );
+
+		if ( 0 === count( $subscriptions ) ) {
+			return $actions;
+		}
+
+		$subscription = reset( $subscriptions );
+
+		foreach ( $links as $link ) {
+			if ( ! \str_contains( $link, 'rcp_sub_details_update_card' ) ) {
+				continue;
+			}
+
+			$new_link = new WP_HTML_Tag_Processor( $link );
+
+			if ( $new_link->next_tag( 'a' ) ) {
+				$new_link->set_attribute( 'href', $subscription->get_mandate_selection_url() );
+			}
+
+			$actions = \str_replace( $link, $new_link, $actions );
+		}
```

Comment:
I think the use of the `WP_HTML_Tag_Processor` indicates that it is not intended this way.

Reply:
Changed in https://github.com/pronamic/wp-pronamic-pay-restrict-content-pro/pull/23/commits/0c46dfe43fc27d2df6800f287e3141988d38fba7.